websites/R/Reddit/presence.ts

```diff
@@ -35,28 +36,56 @@ async function getStrings() {
 	);
 }
 
-const startTimestamp = Math.floor(Date.now() / 1000),
+const presences: { [key in PresenceClients]?: Presence } = {
+		[PresenceClients.Reddit]: presence,
+	},
+	startTimestamp = Math.floor(Date.now() / 1000),
 	oldReddit = document.querySelector("._1tvdSTbdxaK-BnUbzUIqIY") === null;
```

Comment:
Maybe there's a better method for this?
```suggestion
	oldReddit = document.querySelector("header") === null;
```